css/portal-dashboard/num-students.less

```diff
@@ -0,0 +1,23 @@
+@import "./variables";
+
+.numStudentsContainer {
+    width: 168px;
+    height: 30px;
+    border-radius: 15px 0px 0px 15px;
+    background-color: @cc-teal-light7;
+    position: absolute;
+    top: 210px;
+    left: 72px;
+    font-size: 16px;
+    font-family: lato, arial, helvetica, sans-serif;
```

Comment:
I think you can remove this now.  I added the font-family to the top level component so all the children will get it too.